contrib/yaml/sample.yaml

```diff
@@ -0,0 +1,194 @@
+version: 1
```

Comment:
I think `sample.yaml` will be too difficult for most people...

We should have explicit files for each version, so it is a standard copy'n'paste into their installation.

And, there will be different metrics for each version -- especially with 16 and all the `pg_statio_*` views...

Reply:
So what I'm understanding is that you want to keep the previous YAML structure but separate them out into multiple files with the topmost `version` denoting the PostgreSQL version?

Reply:
We can change the YAML structure - if really needed

Reply:
Ideally, people want one file that can over all their PostgreSQL installations, and this is where version support is important.

Reply:
Ok, so according to you, if query's version is not provided, the file's default value will be taken?

Also, so should I rename the topmost `version` to `default_version`?

Reply:
If you give the green light, I'll go ahead with this then!

Reply:
:+1: